render-only-sample/rosumd/RosUmdResource.cpp

```diff
@@ -13,12 +13,13 @@
 #include "RosContext.h"
 
 #include "Vc4Hw.h"
+#include <memory>
```

Comment:
Right now there are conflicts so github won't let me merge it. Can you rebase on top of master? After rebasing, you will get the latest changes and can put this header in precomp.h


    git fetch --all

    git rebase -i upstream/master



Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: marekkedzierski <notifications@github.com>
Sent: Wednesday, March 23, 2016 9:10:44 AM
To: Microsoft/graphics-driver-samples
Cc: Jordan Rhee
Subject: Re: [graphics-driver-samples] XAML textures sampling support (#41)


In render-only-sample/rosumd/RosUmdResource.cpp<https://github.com/Microsoft/graphics-driver-samples/pull/41#discussion_r57187841>:

> @@ -13,12 +13,13 @@
>  #include "RosContext.h"
>
>  #include "Vc4Hw.h"
> +#include <memory>


It will be added during merge - because my branch is so old, that is has no precomp.h in UMD added yet.

-
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/graphics-driver-samples/pull/41/files/78a475253b5655d7e79677e0195c396dc16aad24#r57187841>